root-parent-pom/pom.xml

```diff
@@ -378,7 +378,7 @@
         <profile>
             <id>java11</id>
             <activation>
-                <jdk>[11,16]</jdk>
+                <jdk>[11,17)</jdk>
```

Comment:
From TFM:
```
Note: an upper bound such as ,1.5] is likely not to include most releases of 1.5, since they will have an additional "patch" release such as _05 that is not taken into consideration in the above range.
```